data/json/npcs/refugee_center/surface_staff/Smokes/free_merchant_shopkeep_itemlist.json

```diff
@@ -0,0 +1,188 @@
+[
+  {
+    "type": "item_group",
+    "id": "NC_FREE_MERCHANTS_MERCHANT_worn",
+    "subtype": "collection",
+    "items": [
+      { "item": "boxer_shorts" },
+      { "item": "socks" },
+      { "item": "pants_cargo" },
+      { "item": "tshirt" },
+      { "item": "trenchcoat" },
+      { "item": "boots_steel" },
+      { "item": "firehelmet" },
+      { "item": "duffelbag" },
+      { "item": "bead_bracelet" }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_FREE_MERCHANTS_MERCHANT_carried",
+    "subtype": "collection",
+    "items": [ { "item": "halligan" } ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_FREE_MERCHANTS_MERCHANT_misc",
```

Comment:
Since the free merchant store stock needs some revising anyway, I would suggest separating this out - just use the old evac merchant item lists for now, and in a separate PR we can make a more nuanced new store item group set, including items that spawn around the merchant on shelves and things. I've been doing this for Rubik and it's easy to copy over.

Reply:
Will do.